File: babysitter/src/BabySitter.Core/BabySitters/Models/SitterModel.cs
namespace BabySitter.Core.BabySitters.Models
{
    public class SitterModel
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public long HourlyRate { get; set; }
        public long HourlyRateAfterMidnight { get; set; }
        public long HourlyRateBetweenBedtimeAndMidnight { get; set; }
    }
}

File: babysitter/src/BabySitter.Web/client-app/src/sitters/models/sitter.model.ts
export interface AddSitterModel {
  sitterId?: number;
  firstName?: string;
  lastName?: string;
  hourlyRate?: number;
  hourlyRateBetweenBedtimeAndMidnight?: number;
  hourlyRateAfterMidnight?: number;
}

export interface SitterModel extends AddSitterModel {
  id: number;
}
